src/Workspaces/CSharpTest/EmbeddedLanguages/VirtualChars/CSharpVirtualCharServiceTests.cs

```diff
@@ -83,6 +94,16 @@ public void TestBracesInInterpolatedVerbatimSimpleString()
             Test("$@\"{{\"", "['{',[3,5]]");
         }
 
+        [Fact]
```

Comment:
Thanks!
Link to https://github.com/dotnet/roslyn/issues/29172 would be great.

Reply:
Done!